.env.dev.template

```
@@ -26,5 +35,21 @@ S3_ACCESS_KEY=admin
 S3_SECRET_KEY=admin123
 S3_FORCE_PATH_STYLE=true
 
+# Client dev defaults
+CLIENT_CORE_HOST=http://localhost:8080
+CLIENT_INTRO_COURSE_HOST=http://localhost:8082
+CLIENT_DEVOPS_CHALLENGE_HOST=http://localhost:9000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLIENT_DEVOPS_CHALLENGE_HOST key should go before the CLIENT_INTRO_COURSE_HOST key

(UnorderedKey)